packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -483,15 +483,8 @@ async function handler(
   if (eventType.seatsPerTimeSlot) {
     const booking = await prisma.booking.findFirst({
       where: {
-        OR: [
-          {
-            uid: rescheduleUid || reqBody.bookingUid,
```

Comment:
Removing this looks ok to me, this shouldn't be needed for finding an existing seat/booking